eng/bomgenerator/buildAndRun.cmd

```diff
@@ -0,0 +1,4 @@
+set "versioningClientFileLocation=%~dp0..\versioning\version_client.txt"
```

Comment:
Should this use a scripting language that is OS agnostic?

Reply:
I don't expect this to be run by the pipeline just yet. For now, this is just a convenience for whoever is triggering this manually. At the time of automation - we would rather make this a plugin and move the tool to another place.